contributor-orientation.php

```diff
@@ -396,124 +487,138 @@ function get_pages() {
 			'url'           => 'https://make.wordpress.org/performance/handbook/get-involved/',
 		),
 		'test' => array(
-			'name'          => esc_html__( 'Test', 'wporg' ),
+			'name'          => esc_html__( 'Get started with the Test Team', 'wporg' ),
 			'description'   => esc_html__( 'The Test Team tests, documents, and reports on the WordPress user experience using every device available. Through continuous dogfooding and visual records, the Test Team understands not only what is wrong, but also what is right, while championing user experience.', 'wporg' ),
 			'icon'          => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill-rule="evenodd" d="M17 3c.55 0 1 .45 1 1v10c0 .55-.45 1-1 1H3c-.55 0-1-.45-1-1V4c0-.55.45-1 1-1h14Zm-1 6c-1.6-1.86-3.7-3-6-3S5.6 7.14 4 9c1.6 1.86 3.7 3 6 3s4.4-1.14 6-3Zm-4 0c0-1.1-.9-2-2-2s-2 .9-2 2 .9 2 2 2 2-.9 2-2Zm5 8h-3v-1H6v1H3v1h14v-1Z" clip-rule="evenodd"/></svg>',
 			'url'           => 'https://make.wordpress.org/test/handbook/',
 		),
-	);
+		'hosting' => array(
+			'name'          => esc_html__( 'Get started with the Hosting Team', 'wporg' ),
+			'description'   => esc_html__( 'The Hosting Team works to improve WordPress’ end-user experience across hosting environments through industry collaboration and user education.', 'wporg' ),
+			'icon'          => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><rect x="0" fill="none" width="20" height="20"/><g><path d="M14.9 9c0-.3.1-.6.1-1 0-2.2-1.8-4-4-4-1.6 0-2.9.9-3.6 2.2-.2-.1-.6-.2-.9-.2C5.1 6 4 7.1 4 8.5c0 .2 0 .4.1.5-1.8.3-3.1 1.7-3.1 3.5C1 14.4 2.6 16 4.5 16h10c1.9 0 3.5-1.6 3.5-3.5 0-1.8-1.3-3.3-3.1-3.5z"/></g></svg>',
+			'url'           => 'https://make.wordpress.org/hosting/handbook/get-involved/',
+		),
+		'openverse' => array(
+			'name'        => esc_html__( 'Get started with the Openverse Team', 'wporg' ),
+			'description' => esc_html__( 'The Openverse Team implements new features and new media types; maintains the public API and front-end search engine; and develops WordPress integrations to share Openverse with the entire WordPress community.', 'wporg' ),
+			'icon'        => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><rect x="0" fill="none" width="20" height="20"/><g><path d="M14.9 9c0-.3.1-.6.1-1 0-2.2-1.8-4-4-4-1.6 0-2.9.9-3.6 2.2-.2-.1-.6-.2-.9-.2C5.1 6 4 7.1 4 8.5c0 .2 0 .4.1.5-1.8.3-3.1 1.7-3.1 3.5C1 14.4 2.6 16 4.5 16h10c1.9 0 3.5-1.6 3.5-3.5 0-1.8-1.3-3.3-3.1-3.5z"/></g></svg>',
+			'url'         => 'https://make.wordpress.org/openverse/handbook/new-contributor-guide/'
+		),
+/** 
+* Note: The Plugins, Meta and Tide teams are intentionally not currently displaying as suggested teams because they do not have sufficient new contributor onboarding processes 
+*/			
+		'plugins' => array(
```

Comment:
I don't think this one is technically an issue, but it's easier to read the comment if it's at the same indentation as the rest of the array.

There is also a code style issue here because the comment has some trailing space. If you've got this project set up locally, you can run `npm run lint:php` to check that. A github action could also be added to check this on each PR.

```suggestion

		/**
		 * Note: The Plugins, Meta, and Tide teams are intentionally not currently
		 * displaying as suggested teams because they do not have sufficient new
		 * contributor onboarding processes.
		 */
		'plugins' => array(
```

Reply:
Ah! thank you for updating and the tip about the trailing space!